nft_ingester/src/config.rs

```diff
@@ -68,61 +68,65 @@ pub struct IngesterClapArgs {
     pub parallel_json_downloaders: i32,
 
     #[clap(
-        long("run_api"),
+        long("disable-api"),
         default_value_t = true,
-        env = "IS_RUN_API",
-        help = "Run API (default: false)"
+        action = ArgAction::SetFalse,
+        env = "RUN_API",
+        help = "Disable API (default: true)"
     )]
-    pub is_run_api: bool,
+    pub run_api: bool,
```

Comment:
What's happening here? `disable-api` arg that sets `run_api` @andrii-kl am I missing something?